operator/v1/types_etcd.go

```diff
@@ -43,11 +43,21 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// BackendQuotaGiB sets the etcd backend storage size limit in gibibytes. Defaults to 8GiB.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum=8
+	// +kubebuilder:validation:Maximum=32
```

Comment:
To have a default, you need `omitempty` and two tags
```
// +kubebuilder:default:=8
// +default:=8
```

With omtiempty, I expect:
* Writes without the field should succeed
* Reads will then return the value with the default (which passes validation)
* User writes should be acceptable provided they are within bounds
* Any existing tests that exist with an expected value, will need to be updated to include the new default value where this feature gate is present (eg CustomNoUpgrade tests)

I would hope to see tests for creating both with and without the value initially, and tests for adding the value and trying to increase and decrease the value with the appropriate pass and failure